zone/npc.cpp

```diff
@@ -3239,6 +3239,18 @@ void NPC::AIYellForHelp(Mob *sender, Mob *attacker)
 		}
 
 		float assist_range = (mob->GetAssistRange() * mob->GetAssistRange());
+
+		if (RuleB(Combat, EnableSneakPull) && attacker->sneaking) {
+			assist_range=RuleI(Combat, SneakPullAssistRange);
```

Comment:
Spacing here. Other than that. Looks good! 👍

Reply:
Will squash on merge. So more commits is no biggie.

Reply:
got it